app/grandchallenge/reader_studies/models.py

```diff
@@ -1508,3 +1517,30 @@ def delete(self):
 
     class Meta(RequestBase.Meta):
         unique_together = (("reader_study", "user"),)
+
+
+class AnswerWidgetKindChoices(models.TextChoices):
+    ACCEPT_REJECT = "ACCEPT_REJECT", "Accept/Reject Findings"
+
+
+class AnswerWidget(models.Model):
```

Comment:
I think `QuestionWidget` would make more sense as these are widgets for questions, not answers. Using `question.widget` would make more sense than `question.answer_widget`.